File: modelAPI/Baidu/ChatGLM.py
from .BaiduAPI import BaiduAPI

class ChatGLM(BaiduAPI):
    def __init__(self):
        super().__init__()
        self.url = "https://aip.baidubce.com/rpc/2.0/ai_custom/v1/wenxinworkshop/chat/chatglm2_6b_32k?access_token=" + self.access_token
    

File: modelAPI/Baidu/ChatGLM.go
package Baidu

const CHATGLM = "https://aip.baidubce.com/rpc/2.0/ai_custom/v1/wenxinworkshop/chat/chatglm2_6b_32k?access_token="

type ChatGLM struct {
	Baidu
	url string `https://aip.baidubce.com/rpc/2.0/ai_custom/v1/wenxinworkshop/chat/chatglm2_6b_32k?access_token=`
}

func NewChatGLM() *ChatGLM {
	baidu := NewBaidu()
	accessToken, _ := baidu.getAccessToken()
	baidu.url = CHATGLM + accessToken
	chatGLM := &ChatGLM{
		Baidu: *baidu,
	}
	return chatGLM
}
